pkg/overlay/cache.go

```diff
@@ -43,34 +44,26 @@ type DB interface {
 	SelectNewStorageNodes(ctx context.Context, count int, criteria *NewNodeCriteria) ([]*pb.Node, error)
```

Comment:
The offline check needs to be part of select nodes query.

Reply:
which offline check exactly?

Reply:
previously `SelectNewStorageNodes` worked on overlay which didn't contain nodes that were offline for the last ping. So it shouldn't select nodes where the last ping failed.

Reply:
This was already done in a previous PR: https://github.com/storj/storj/blob/master/satellite/satellitedb/overlaycache.go#L42-L43

Reply:
ah, nevermind then... missed that change